borsh/src/ser/mod.rs

```diff
@@ -94,6 +94,31 @@ impl_for_integer!(u32);
 impl_for_integer!(u64);
 impl_for_integer!(u128);
 
+mod non_zero {
+    use super::*;
+    use std::num::*;
```

Comment:
To support no-std context, we should use `num` from `core` instead of `std`:

```suggestion
    use core::num::*;
```